packages/components/psammead-social-embed/src/Canonical/index.jsx

```diff
@@ -47,6 +48,11 @@ export const providers = {
         window.twttr.widgets.load();
       }
     },
+    onSdkLoad: onRender => {
+      window.twttr.ready(twttr => {
+        twttr.events.bind('rendered', onRender);
```

Comment:
So just for my understanding, the `onRender` function will fire once the twitter embed has finished rendering, by firing on the rendered event that the twitter embed emits? And within the `onRender` function we can do some form of styling changes after the twitter widget has done everything it needs to do?

Reply:
Thanks @HarveyPeachey, that's correct 🙂

Reply:
> So just for my understanding, the `onRender` function will fire once the twitter embed has finished rendering, by firing on the rendered event that the twitter embed emits? And within the `onRender` function we can do some form of styling changes after the twitter widget has done everything it needs to do?

Yes that's correct! We chose this method because I believe without `onRender` and the `isLoaded `state, performing the required styling changes degraded the page quality. Hope that makes sense.

Reply:
Yeah so atm the social embeds have a `min-height` of 300px, so if a tweet has a height lower than 300px it will have more spacing beneath it, this PR will allow us to remove that additional spacing by altering styles on render 👍

Reply:
That makes sense, it might be beneficial adding this use case in the readme for more context or adding a story to show it in action?

Reply:
I think the intention was to update the readme, just hasn't been actioned yet but good point!